sentryflow-clients/.gitignore

```diff
@@ -0,0 +1,27 @@
+### macOS
+.DS_Store
```

Comment:
will the .gitignore file in the root directory won't work?

Reply:
You're right, thanks for pointing out this.